tabbycat/api/views.py

```diff
@@ -411,7 +414,6 @@ def get_queryset(self):
             Prefetch(
                 typ.__name__.lower() + "_set",
                 queryset=typ.objects.all().select_related('question', 'question__tournament'),
-                to_attr=typ.__name__,
```

Comment:
Removing this line reduced the number of SQL queries for a 70-feedback GET from to 1328 to 8. I now suspect that when I had suggested that paging be enabled by default with sensible limits, the actual symptom I was experiencing wasn't too many objects, but just a N+1 SQL query issue. I still think pagination should be enabled by default, but just wanted to say that it's not the exact issue I was having.

Reply:
Actually… I might go back on myself with the pagination thing, let me explore this a little first.